config/default/manager_auth_proxy_patch.yaml

```diff
@@ -15,11 +15,18 @@ spec:
             - "--secure-listen-address=0.0.0.0:8443"
             - "--upstream=http://127.0.0.1:8080/"
             - "--logtostderr=true"
-            - "--v=10"
+            - "--v=0"
           ports:
             - containerPort: 8443
               protocol: TCP
               name: https
+          resources:
+            limits:
+              cpu: 500m
+              memory: 128Mi
+            requests:
+              cpu: 5m
+              memory: 64Mi
```

Comment:
Same comment as above. The limits will need to changed here and the bundle command reran to pick up the change in the manifests.

Reply:
I'm not sure I understand. The change was made in this file and applied to the bundle with `make bundle`, which manifests are you referring to?